config.json

```
@@ -0,0 +1,58 @@
+{
+    "bellecour": {
+        "STARTBLOCK": 4543300,
```

```suggestion
        "START_BLOCK": 4543300,
```
And other occurences.